tgui/packages/tgui/interfaces/NtosMessenger.js

```diff
@@ -51,7 +51,7 @@ export const NtosMessenger = (props, context) => {
   if (viewing_messages) {
     return (
       <NtosWindow width={400} height={600}>
-        <NtosWindow.Content>
+        <NtosWindow.Content scrollable >
```

Comment:
space after scrollable >:(

Reply:
I thought I had fixed that, but I think I hit Ctrl+Z by accident
*becomes dust*